src/apps/virtio_net/virtio_net.lua

```diff
@@ -0,0 +1,76 @@
+-- Application to connect to a virtio-net driver implementation
+--
+-- Licensed under the Apache 2.0 license
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Copyright (c) 2015 Virtual Open Systems
```

Comment:
note: The *actual* copyright holder is determined by a contract between the author and that other party. Writing this in the source file does not make it so. In practice these lines are often wrong e.g. in OpenStack many people write "Copyright OpenStack Foundation" because they think the contributor agreement includes copyright assignment when it actually does not (and so copyright probably belongs to their employer as a matter of fact).

Ugh.